web-sdk/v3.md

```diff
@@ -628,3 +628,156 @@ Here are some examples of what is achievable using the customization:
     { buttonText: 'Theme 2', slug: 'OJbxZzW', title: 'V3 Fidel Web SDK' },
     { buttonText: 'Theme 3', slug: 'eYBGrrQ', title: 'V3 Fidel Web SDK' },
 ]} />
+
+## Card entity
+
+This object is returned through the `onCardEnrolledCallback` when a card is successfully enrolled.
+
+<table>
+<thead>
+  <tr>
+    <th>Parameter</th>
+    <th>Value</th>
+    <th>Description</th>
+</thead>
+<tbody>
+  <tr>
+    <td><code>id</code></td>
+    <td>string</td>
+    <td>ID of the enrolled card</td>
+  </tr>
+  <tr>
+    <td><code>accountId</code></td>
+    <td>string</td>
+    <td>Account ID of the program where the card was enrolled</td>
+  </tr>
+  <tr>
+    <td><code>programId</code></td>
+    <td>string</td>
+    <td>ID of the program where the card was enrolled</td>
+  </tr>
+  <tr>
+    <td><code>scheme</code></td>
+    <td>string<br>Possible values: 'mastercard', 'visa', 'amex'</td>
+    <td>Card scheme</td>
+  </tr>
+  <tr>
+    <td><code>lastNumbers</code></td>
+    <td>number</td>
+    <td>Last 4 numbers of the card</td>
+  </tr>
+  <tr>
+    <td><code>firstNumbers</code></td>
+    <td>number</td>
+    <td>First 6 numbers of the card</td>
+  </tr>
+  <tr>
+    <td><code>expYear</code></td>
+    <td>number</td>
+    <td>First 6 numbers of the card</td>
+  </tr>
+  <tr>
+    <td><code>expYear</code></td>
+    <td>Number</td>
+    <td>Expiration year of the card</td>
+  </tr>
+  <tr>
+    <td><code>expMonth</code></td>
+    <td>Number</td>
+    <td>Expiration month of the card</td>
+  </tr>
+  <tr>
+    <td><code>expDate</code></td>
+    <td>String</td>
+    <td>Expiration date (MM/YY format)</td>
+  </tr>
+  <tr>
+    <td><code>created</code></td>
+    <td>string</td>
+    <td>Card enrollment date</td>
+  </tr>
+  <tr>
+    <td><code>countryCode</code></td>
+    <td>String</td>
+    <td>Card country code</td>
+  </tr>
+  <tr>
+    <td><code>termsOfUse</code></td>
+    <td>Boolean</td>
+    <td>Accepted or declined terms of use</td>
+  </tr>
+  <tr>
+    <td><code>live</code></td>
+    <td>Boolean</td>
+    <td>Flag that checks if the card is from a live or a test environment</td>
+  </tr>
+  <tr>
+    <td><code>metadata</code></td>
+    <td>Object</td>
+    <td>Returns the metadata set by the enrollment API call</td>
+  </tr>
+</tbody>
+</table>
+
+## Error object
+
+This object is returned whenever you receive an error callback.
+
+<table>
+<thead>
+  <tr>
+    <th>Parameter</th>
+    <th>Value</th>
+    <th>Description</th>
+</thead>
+<tbody>
+  <tr>
+    <td><code>cardId</code></td>
+    <td>string (optional)</td>
+    <td>ID of the enrolled card. Might not be present if card is not enrolled at the error stage</td>
```

Comment:
@cornelfd Thanks Cornel. This is a generic error object, and when the card is already enrolled it will also contain the cardId. For example, if the error is thrown in the card enrollment phase, it will not have a cardId since the card is not enrolled yet. Errors on verification phase will contain a cardId.

Reply:
I think it would be useful to document what you explained to me above then 😃 The last phrase is formulated in a more ambiguous way. I think we were clear enough in your explanation in response to my comment.